src/index.html

```diff
@@ -9,10 +9,64 @@
     <title>Product cards</title>
     <link
       rel="stylesheet"
-      href="./styles/index.scss"
+      href="./styles/index.css"
+    />
+    <link
+      href="https://fonts.googleapis.com/css2?family=Roboto:wght@400;500;700&display=swap"
+      rel="stylesheet"
     />
   </head>
   <body>
-    <h1>Product cards</h1>
+    <article
+      class="product-card"
+      data-qa="card"
+    >
+      <div class="product-card__wrapper">
+        <header class="product-card__header">
+          <img
+            src="images/imac.jpeg"
+            alt="APPLE A1419 iMac 27 Retina 5K Monoblock (MNED2UA/A)"
+            class="product-card__img"
+          />
+        </header>
+        <section class="product-card__main product-info">
+          <div class="product-info__titles">
+            <h3 class="product-info__title">
+              APPLE A1419 iMac 27" Retina 5K Monoblock (MNED2UA/A)
+            </h3>
+            <h5 class="product-info__subtitle">Product code: 195434</h5>
+          </div>
+          <div class="reviews">
+            <div class="stars stars--4">
+              <div class="stars__star"></div>
+              <div class="stars__star"></div>
+              <div class="stars__star"></div>
+              <div class="stars__star"></div>
+              <div class="stars__star"></div>
+            </div>
+            <div class="reviews__block">
+              <p class="reviews__text">Reviews: 5</p>
+            </div>
+          </div>
+          <div class="product-info__block">
+            <div class="product-info__item">
+              <p class="product-info__text">Price:</p>
+            </div>
+            <div class="product-info__item">
+              <h3 class="product-info__price">$2,199</h3>
+            </div>
```

Comment:
Do you really need this `<div class="product-info__item">` wrapper here

Reply:
Better safe than sorry :)